src/components/Tabs/Tabs.tsx

```diff
@@ -194,7 +195,7 @@ const Tabs = (props: TabsProps): JSX.Element => {
   }, [selectedTab]);
 
   return (
-    <div className={'bsds-tabs' + (hasOverflow ? ' has-overflow' : '')}>
+    <div className={`bsds-tabs ${hasOverflow ? ' has-overflow' : ''} ${props.className || ''}`}>
```

Comment:
when i open the PR's SB link and try adding `bsds-mt-8x` to tabs, it isn't applying a top margin